extension/src/time_weighted_average.rs

```diff
@@ -548,6 +538,44 @@ pub fn time_weighted_average_interpolated_integral<'a>(
     time_weighted_average_integral(target, unit)
 }
 
+#[pg_operator(immutable, parallel_safe)]
+#[opname(->)]
+pub fn arrow_time_weighted_average_interpolated_integral<'a>(
+    tws: Option<TimeWeightSummary<'a>>,
+    accessor: TimeWeightInterpolatedIntegralAccessor<'a>,
+) -> Option<f64> {
+    let prev = if accessor.flags & 1 == 1 {
+        Some(accessor.prev.clone().into())
+    } else {
+        None
+    };
+    let next = if accessor.flags & 2 == 2 {
+        Some(accessor.next.clone().into())
+    } else {
+        None
+    };
+
+    // Convert from num of milliseconds to DurationUnit and then to string
```

Comment:
This section of code is a workaround for the DurationUnit enum not working with the flat_serialize macro.